packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenters/add/upgrade-range/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dedicated_cloud_upgrade_range": "Migration"
```

Comment:
Sibling translation files are using a different key:
`managed_baremetal_user_rights` instead of `dedicated_cloud_upgrade_range`.

Is it normal?